### ISSUE TITLE
Need a way stop AR on a-scene unloading

### ISSUE THREAD

**Issue by paztis** [artoolkit/jsartoolkit5#73]
Currently, when we remove the <a-scene> dom element, all 3d is stopped.
But the video and the overlay helpers (markersAreaEnabled and 
trackingBackend) are still present.
If I restart again the AR, new overlay helpers arppears near the first one.

Is there a way to stop properly the AR (ie stop video, remove overlays, destroy ARController, ...) ? I really need it to display/hide the AR in an applicaiton lifetime.

Thanks

**Comment by ThorstenBux** [comment]
isn't this working for you? https://github.com/artoolkit/jsartoolkit5/issues/72

**Comment by ThorstenBux** [comment]
besides that there is:
`        this.arController.dispose();`

**Comment by paztis** [comment]
hi,

I was tryion to access to the ARCotroller without success on my code (because it's cxreated automatically by the a-frame <a-scene> tag.

But I just found an accessor : 
`aSceneEl.systems.arjs._arSession.arContext.arController.dispose()`

Hard to access it ^^
I try to access to the video also now.

But hte graphic helpers are still on the screen.

The library is not really planned to do this. Everything is accessible to start AR, but then nothing during its lifetime...

**Comment by ThorstenBux** [comment]
I agree that everything is there to startAR but nothing to stop it. 
The other thing is that you are using arjs and not plainly jsartoolkit. You might consider asking @jeromeetienne (who is the author of arjs) for a solution.

**Comment by paztis** [comment]
yes sorry. I ask him
Thanks
